specifyweb/specify/models.py

```diff
@@ -7462,3 +7486,97 @@ class Meta:
 
     timestamptracker = FieldTracker(fields=['timestampcreated', 'timestampmodified'])
     save = partialmethod(custom_save)
+
+class CollectionObjectType(models.Model):
+    specify_model = datamodel.get_table('collectionobjecttype')
+
+    # ID Field
+    id = models.AutoField(primary_key=True, db_column='CollectionObjectTypeID')
+
+    # Fields
+    name = models.CharField(blank=False, max_length=255, null=False, unique=False, db_column='Name', db_index=False)
+    isloanable = models.BooleanField(blank=True, null=True, unique=False, db_column='IsLoanable', db_index=False)
```

Comment:
[In a meeting Jul 3 2024](https://docs.google.com/document/d/1CYC98mfILF91djuksaqwf-6Vd9TR5xqSTYMrTgkmnE8/edit#heading=h.t7ng7ufxkjaw), the 'loanability' of a CollectionObjectType was discussed. 
Specifically, it was determined that `isloanable` should be removed from the CollectionObjectType table. 

Was it decided somewhere else to keep this field?

Reply:
It will be removed